src/Viewer/components/LeftPanel/DownloadHelper.js

```diff
@@ -31,8 +31,8 @@ const downloadCompressedFile = () => {
     //  which avoids interruption of uncompressed logs download
     link.target = "_blank";
 
-    const urlParams = new URLSearchParams(window.location.search);
-    const filePath = urlParams.get("filePath");
+    let filePath = window.location.search.split("filePath=")[1];
```

Comment:
Seems like we're using this logic in three places. Can we extract it into a method?